src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -50,13 +51,37 @@ object UserAuth {
 }
 
 object UserAuthJsonSupport extends JsonSupport {
-  implicit val RawlsUserEmailFormat = jsonFormat1(RawlsUserEmail)
```

Comment:
these format changes make these "value" objects look like strings
```
{ "userSubjectId": "123", "userEmail": "foo@bar.com" }
```
instead of
```
{ "userSubjectId": {"value": "123"}, "userEmail": {"value": "foo@bar.com"} }
```